package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "examples",
+  "version": "0.0.0-dev",
+  "description": "Examples repo",
+  "main": "package.json",
+  "scripts": {
+    "test": "exit 0",
+    "start": "tsx scripts/start.ts",
+    "build": "npm run build --workspaces"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/romcal/romcal-examples.git"
+  },
+  "keywords": [
+    "examples",
+    "romcal",
+    "react",
+    "html",
+    "iife",
+    "rest-api"
+  ],
+  "author": "emangier",
```

Maybe you should add youself here, alomg with the email addresses (like [we have it](https://github.com/romcal/romcal/blob/1d928b7b7aa65b77299cb2992a39c8aea686a4f2/package.json#L25-L41) in the `romcal` repo).


```suggestion
  "author": {
    "name": "Etienne Magnier",
    "email": "etienne.magnier@gmail.com",
    "url": "https://github.com/emagnier"
  },
  "contributors": [
    {
      "name": "your name/pseudonym",
      "email": "your email address",
      "url": "https://github.com/TobiTenno or https://matej.voboril.dev"
    }
  ],
```

---

i'm not in the romcal one, funny enough. i'll update this from your suggestion start, though.

---

```suggestion
  "author": {
    "name": "Etienne Magnier",
    "email": "etienne.magnier@gmail.com",
    "url": "https://github.com/emagnier"
  },
  "contributors": [
    {
      "name": "Matej Voboril",
      "email": "tobiah@pm.me",
      "url": "https://github.com/TobiTenno"
    },
    {
      "name": "Tukusej’s Sirs",
      "email": "tukusejssirs@protonmail.com",
      "url": "https://github.com/tukusejssirs"
    }
  ],
```

---

> i'm not in the romcal one, funny enough. i'll update this from your suggestion start, though.

I agree you should add yourself in the romcal repo too. :pray: